internal/verifier/check.go

```diff
@@ -85,9 +85,12 @@ func (verifier *Verifier) CheckWorker(ctx context.Context) error {
 
 	for {
 		select {
-		case err := <-verifier.changeStreamErrChan:
+		case err := <-verifier.srcChangeStreamReader.ChangeStreamErrChan:
 			cancel()
-			return errors.Wrap(err, "change stream failed")
+			return errors.Wrapf(err, "got an error from %s", verifier.srcChangeStreamReader)
```

Comment:
Do you prefer “got an error” to “failed” here? The latter seems (to me, at least) like stronger, more concise wording.

Maybe:
```
errors.Wrapf(err, "%s failed", verifier.srcChangeStreamReader)
```

Reply:
Agreed.